package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@eppo/js-client-sdk-common",
-  "version": "4.7.0",
+  "version": "4.7.0-alpha.0",
```

Comment:
4.7.0 hasn't been released yet. I'm thinking we could do a pre-release first before officially bumping the minor version to 7

Reply:
To release 4.7.0-alpha.1

1. Update the md5 hasher to not expect a byte string salt `.appendBinary` --> `.append` when it is safe to do so
2. And publish locally with `make prepare` and `yarn version --prerelease --preid alpha` and `yarn publish --tag alpha` <-- Please double check the docs that these commands make sense